### ISSUE TITLE
Does this require data credits?

### ISSUE THREAD

**Issue by shawaj** [gradoj/discoclient#5]
Does this go over Helium network and require data credits?

**Comment by gradoj** [comment]
Yes, the disco client running on each hotspot is a sensor in the console. It sends lorawan packets. I've just been paying for the DC the credit isn't implemented. 

Be warned this is shutting down with Heroku free tier this month. If you still want to try DM me on discord to enable your sensor. I have disabled by default cause someone was gaming my dc

**Comment by shawaj** [comment]
@gradoj this is Aaron from Nebra. We were thinking to implement on our units. Can you ping me on aaron@nebra.com?
